tensorflow/lite/experimental/swift/BUILD.apple

```diff
@@ -17,7 +17,7 @@ swift_library(
     tags = TFL_DEFAULT_TAGS,
     visibility = ios_visibility_whitelist(),
     deps = [
-        "//tensorflow/lite/experimental/ios:TensorFlowLiteC",
+        "//tensorflow/lite/experimental/ios:tensorflow_lite_c",
```

Comment:
Done, FYI I added them to the swift_library and not the cc_library because of https://github.com/bazelbuild/rules_swift/issues/432

Reply:
Thanks! Do you know if this should be applied to TensorFlowLiteC_framework target, too?

Reply:
In the current scheme it doesn't make a difference because no linking target (at least in this public repo) depends on it. But I added it for completeness since it doesn't hurt!